File: our_story_front/lib/features/pairing/data/models/pairing_status.dart
enum PairingStatus {
  pending,
  used,
  expired,
  cancelled;

  static PairingStatus fromString(String value) {
    switch (value.toUpperCase()) {
      case 'PENDING':
        return PairingStatus.pending;
      case 'USED':
        return PairingStatus.used;
      case 'EXPIRED':
        return PairingStatus.expired;
      case 'CANCELLED':
        return PairingStatus.cancelled;
      default:
        return PairingStatus.pending;
    }
  }
}

File: our-story-back/src/main/java/com/ourstory/our_story_back/enums/PairingStatus.java
package com.ourstory.our_story_back.enums;

public enum PairingStatus {
  PENDING,    // Código generado, esperando verificación
  USED,       // Código usado exitosamente
  EXPIRED,    // Código expirado (24 horas)
  CANCELLED   // Solicitud cancelada por el usuario
}
